src/asset_folder_importer/asset_folder_sweeper/find_files.py

```diff
@@ -0,0 +1,71 @@
+import os
+from pprint import pprint
+import logging
+import mimetypes
+from asset_folder_importer.asset_folder_sweeper.posix_get_mime import posix_get_mime
+import re
+
+def find_files(cfg,db):
+    #Step three. Find all relevant files and bung 'em in the database
+    startpath = cfg.value('start_path',noraise=False)
+
+    print "Running from '%s'" % startpath
+
+    #mark a file as to be ignored, if any of these regexes match. This will prevent vsingester from importing it.
+    pathShouldIgnore = [
+        'Adobe Premiere Pro Preview Files',
+        'System Volume Information',
+        '\.TMP',
+        '\/\.',  #should ignore a literal dot, but only if it follows a /
+        '\.pk$',
+        '\.PFL$',
+        '\.PFR$',   #Cubase creates these filetypes
+        '\.peak$',
+        '_synctemp', #this is created by PluralEyes
+    ]
+
+    reShouldIgnore = []
+    for expr in pathShouldIgnore:
+        reShouldIgnore.append(re.compile(expr))
+
+    #try:
+    n=0
+    for dirpath,dirnames,filenames in os.walk(startpath):
+        #print dirpath
+        #pprint(filenames)
+        for name in filenames:
+            if name.startswith('.'):
+                continue
+            shouldIgnore = False
+
+            for regex in reShouldIgnore:
+                if regex.search(dirpath) is not None:
+                    shouldIgnore = True
+
```

Comment:
I am not sure what you want me to do here. Is the fact that the value of reShouldIgnore is supplied by the function called re_should_ignore and that this code sits in the find_files function enough?

@fredex42